Frameworks/AddressBook/ABContactInternal.mm

```diff
@@ -28,4 +28,18 @@ - (id)initWithContact:(WACContact*)contact {
     return self;
 }
 
++ (NSDate*)convertDate:(WACContactDate*)date {
```

Comment:
`-[NSDate _dateWithWACContactDate:]` sounds perfect to me!

---
In reply to: [75170063](https://github.com/Microsoft/WinObjC/pull/884#discussion_r75170063) [](ancestors = 75170063,75166689,75164209,75146840)